server/handler.go

```diff
@@ -0,0 +1,258 @@
+package server
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/ASeegull/SmartFridge/server/database"
+	"github.com/gorilla/websocket"
+	log "github.com/sirupsen/logrus"
+
+	pb "github.com/ASeegull/SmartFridge/protoStruct"
+)
+
+//mock data
+const (
+	defaultHearBeat = 3
+	adminID         = "9079744c-ab87-4083-8400-19c14628c26f"
+)
+
+func sendErrorMsg(w http.ResponseWriter, err error, status int) {
+	log.Debug(err)
```

Comment:
This error will be logged twice

Reply:
But I don't use logger before using this method

Reply:
Lets look at example:
```
+	foods, err := userID.GetFoodsInFridge()
 +	if err != nil {
 +		sendErrorMsg(w, err, http.StatusInternalServerError)
 +		return
 +	}
```
Both GetFoodsInFridge and sendErrorMsg will log error
It is not critical comment at all, but pay attantion